### DISCUSSION TITLE
v2.226

### DISCUSSION THREAD

**Discussion by anddea** [anddea/revanced-patches#654] [Announcements]
> [!WARNING]
> If the ReVanced Extended menu in settings is missing, then clear patch options / `options.json`.

You can check [unique features](https://github.com/anddea/revanced-patches/discussions/446).

**Comment by 54m4d** [comment]
Is it possible to hide the search suggestions that appear when we tap the search icon?

**Reply by anddea** [reply]
I'll look into it later, previously, the old method stopped working.

**Reply by 54m4d** [reply]
👍🏼
